pages/src/_data/workerinfo.yml

```diff
@@ -1,5 +1,6 @@
 alchemist:
   name: Alchemist
+  plural: Alchemists
   traits:
     primary: Dexterity?
```

Comment:
Can we confirm these? I had put question marks because I wrote this before they were out/publicly known.

Reply:
I can look at it tomorrow morning if those are actually what it says in the game aswell

Reply:
I just checked, it's right. I was hoping you had your game open already lol